<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditEtudiant</class>
 <widget class="QDialog" name="DialogEditEtudiant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>213</width>
     <height>236</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="text">
       <string>Abandonner</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pushButtonEdit">
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QComboBox" name="comboBoxOption"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="labelOption">
      <property name="text">
       <string>Nouvelle option</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEditPrenom"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEditNom"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="labelPrenom">
      <property name="text">
       <string>Nouveau Pr√©nom</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="labelNom">
      <property name="text">
       <string>Nouveau nom</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="labelErrorMessage">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
